front/src/common/authorization/roleBaseAccessControl.ts

```diff
@@ -21,38 +21,17 @@ export type RequiredUserRolesFor = {
 export const REQUIRED_USER_ROLES_FOR: RequiredUserRolesFor = {
```

Comment:
Is this struct still useful? It looks like we can get rid of this and check directly user rôle.

Reply:
Can this work be done in a follow up PR? Since paced trains endpoints are being created rn it's probably better if they have the new role set early (and should any of these new endpoints be merged, it breaks this PR...).